packages/rect/examples/basic.example.js

```diff
@@ -8,7 +8,7 @@ export let Example = () => (
     {({ ref, rect }) => (
       <div>
         <pre>{JSON.stringify(rect, null, 2)}</pre>
-        <textarea>resize this</textarea>
```

Comment:
fixes: `Warning: Use the `defaultValue` or `value` props instead of setting children on <textarea>.`